extract_parse.sh

```diff
@@ -11,16 +11,16 @@ apiKey="$1"
 func="$2"
 file_path="$3"
 
-upload
 if [ "$func" == "extract" ]; then
+    upload
```

Comment:
qq: should upload out of the if, elif, so upload will also `apply` for parse logic?

Reply:
nvm. It looks like this is a bug in previous code.